jms/1.7/modules/ROOT/pages/jms-consume.adoc

```diff
@@ -70,27 +63,62 @@ The syntax for consuming messages from a Topic is:
 </jms:consume>
 ----
 
-All the `consume` operation parameters remain the same, but now you can handle the subscription configuration in the `topic-consumer` parameter.
 
-For details on how to configure the Topic subscriptions go to xref:jms-topic-subscription.adoc[Using Topic Subscriptions].
+== Wait for a Message
 
+By default, the *Consume* operation has a maximum wait time configured of `10000` milliseconds, which produces a `JMS:TIMEOUT` error if no message is available in that period.
+In order to configure a timeout that fits better your use case, configure the *Maximum wait* and *Maximum wait unit* fields.
+To wait indefinitely for a message to arrive, set the *Maximum wait* field to `-1`, which prevents a `JMS:TIMEOUT` error.
 
-== Incoming Message Metadata
+== Poll for a Message
 
-As stated before, each Message received will consist of two parts:
+When polling for a message with a certain frequency, you don't want the consumer to block waiting for the message, but instead just try to consume it if one is present.
```

Comment:
"you" are not polling for a message, so this needs rewording to clarify, something like...:
```suggestion
To prevent message polling from blocking other actions as it waits for a message, set the *Maximum wait* field to `0`. This setting prevents both blocking and a timeout error. If polling produces no results, the *Consume* operation output is `null`.
```